spec/features/data_migrate_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "spec_helper"
+
+RSpec.describe 'DataMigrate' do
+  let(:gemfile) { IO.read("#{project_path}/Gemfile") }
+  let(:annotate_task) { IO.read("#{project_path}/lib/tasks/auto_annotate_models.rake") }
+
+  before(:all) do
```

Comment:
Beware of using before(:all) as it may cause state to leak between tests. If you are using rspec-rails, and use_transactional_fixtures is enabled, then records created in before(:all) are not automatically rolled back.